src/dapps/DappShortcutsRewards.tsx

```diff
@@ -0,0 +1,172 @@
+import { BigNumber } from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import Animated from 'react-native-reanimated'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { useSelector } from 'react-redux'
+import Button, { BtnSizes } from 'src/components/Button'
+import { convertDollarsToLocalAmount } from 'src/localCurrency/convert'
+import {
+  getLocalCurrencySymbol,
+  localCurrencyExchangeRatesSelector,
+} from 'src/localCurrency/selectors'
+import { positionsWithClaimableRewardsSelector } from 'src/positions/selectors'
+import { ClaimablePosition } from 'src/positions/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { Currency } from 'src/utils/currencies'
+
+function DappShortcutsRewards() {
+  const { t } = useTranslation()
+  const insets = useSafeAreaInsets()
+
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(localCurrencyExchangeRatesSelector)
+  const positionsWithClaimableRewards = useSelector(positionsWithClaimableRewardsSelector)
+
+  const handleClaimReward = (position: ClaimablePosition) => () => {
+    // do something
+  }
+
+  const renderItem = ({ item }: { item: ClaimablePosition }) => {
+    let claimableFiatValue = new BigNumber(0)
+    item.claimableShortcut.claimableTokens.forEach((token) => {
+      claimableFiatValue = claimableFiatValue.plus(
+        BigNumber(token.priceUsd).times(BigNumber(token.balance))
+      )
+    })
+    const claimableValueLocalCurrency = convertDollarsToLocalAmount(
+      claimableFiatValue,
+      localCurrencyExchangeRate[Currency.Dollar]
+    )
+
+    return (
+      <View style={styles.card} testID="DappShortcutsRewards/Card">
+        <View style={styles.rewardInfoContainer}>
+          <View style={styles.rewardAmountContainer}>
+            <Text style={styles.rewardLabel}>
+              {t('dappShortcuts.claimRewardsScreen.rewardLabel')}
+            </Text>
+
+            <Text style={styles.rewardAmount}>
+              {item.claimableShortcut.claimableTokens
+                .map((token) => `${new BigNumber(token.balance).toFixed(5)} ${token.symbol}`)
+                .join(', ')}
+            </Text>
+            {claimableValueLocalCurrency && (
+              <Text style={styles.rewardFiatAmount}>
+                {`${localCurrencySymbol} ${claimableValueLocalCurrency.toFixed(
+                  claimableValueLocalCurrency.lt(0.01) ? 5 : 2
+                )}`}
+              </Text>
+            )}
```

Comment:
Could we use our existing `TokenDisplay` component instead?
I'd prefer we avoid custom formatting whenever possible.

Reply:
yes that makes sense, i made the change!